synapseutils/sync.py

```diff
@@ -558,14 +560,17 @@ def __init__(
         """
         self._syn = syn
 
-        max_concurrent_file_transfers = max(
-            int(max_concurrent_file_transfers or self._syn.max_threads / 2), 1
+        # TODO: Extract this to a shared location that is further up the stack
+        self._max_concurrent_file_transfers = max(
+            int(max_concurrent_file_transfers or self._syn.max_threads), 1
```

Comment:
why not `max_threads * 2` as it is [here](https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1084)

Reply:
https://github.com/Sage-Bionetworks/synapsePythonClient/pull/1084

Will be getting merged into this draft